inlong-manager/manager-service/src/main/java/org/apache/inlong/manager/service/group/compromize/Compromizer.java

```diff
@@ -0,0 +1,27 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements. See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License. You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.apache.inlong.manager.service.group.compromize;
+
+/**
+ * Operator which ensure the consistency of the state of all components within the group
+ */
+public interface Compromizer {
+
+    public abstract void compromize(String inlongGroupId);
```

Comment:
1. Non-default methods in interfaces do not require the `public abstract` modifier.
2. Please add javadoc for the method in interfaces, thanks.

![image](https://github.com/apache/inlong/assets/31994335/9467d5cd-1e05-49e7-9be9-6464b409d39c)

Reply:
done, removed